app/controllers/password_resets_controller.rb

```diff
@@ -0,0 +1,40 @@
+require 'bcrypt'
```

Comment:
This (I think) requires the system to have bcrypt installed in some way, maybe directly installed or through a dependency. But, your app now depends on it since you call it. 

Instead of requiring it, what do you think of adding it to your Gemfile, and then rails will autoload it?

Reply:
Yep that works!